frontend/src/components/Checklists.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { Fragment, useContext } from 'react';
+import {
+  Typography,
+  List,
+  ListItem,
+  Box,
+  Chip,
+  Checkbox,
+} from '@material-ui/core';
+import { Module } from '../types';
+import { AppContext } from '../context';
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+import PanoramaFishEyeIcon from '@material-ui/icons/PanoramaFishEye';
+import { useStyles, } from '../styles';
+import ReactMarkdown from 'react-markdown';
+import {
+  getSupportedTools,
+  getSelectedTools,
+} from '../utils/moduleHelpers';
+
+interface Props {
+  module: Module;
+  readOnlyMode: boolean;
+}
+interface ChecklistProps {
+  toolsSupported: string[];
+  readOnlyMode: boolean;
+}
+
+const ListoCheckbox = (props: ChecklistProps) => {
+  const classes = useStyles();
+  if (props.toolsSupported.length > 0) {
```

Comment:
Due to falsy checks you can just test `if (props.toolsSupported.length)` if you want to check if the array has elements